tgui-next/packages/tgui/interfaces/Pandemic.js

```diff
@@ -0,0 +1,296 @@
+              </LabeledList>
+            </Grid.Column>
+          </Grid>
+          {!!virus.is_adv && (
+            <Fragment>
+              <Section
+                title="Statistics"
+                level={2} >
+                <Grid>
+                  <Grid.Column>
+                    <LabeledList>
+                      <LabeledList.Item label="Resistance">
+                        {virus.resistance}
+                      </LabeledList.Item>
+                      <LabeledList.Item label="Stealth">
+                        {virus.stealth}
+                      </LabeledList.Item>
+                    </LabeledList>
+                  </Grid.Column>
+                  <Grid.Column>
+                    <LabeledList>
+                      <LabeledList.Item label="Stage speed">
+                        {virus.stage_speed}
+                      </LabeledList.Item>
+                      <LabeledList.Item label="Transmissibility">
+                        {virus.transmission}
+                      </LabeledList.Item>
+                    </LabeledList>
+                  </Grid.Column>
+                </Grid>
+              </Section>
+              <Section
+                title="Symptoms"
+                level={2} >
+                {symptoms.map(symptom => (
+                  <Collapsible
+                    key={symptom.name}
+                    title={symptom.name} >
+                    <Section>
+                      <PandemicSymptomDisplay symptom={symptom} />
+                    </Section>
+                  </Collapsible>
+                ))}
+              </Section>
+            </Fragment>
+          )}
+        </Section>
+      );
+    })
+  );
+};
+
+export const PandemicSymptomDisplay = props => {
+  const { symptom } = props;
+  const {
+    name,
+    desc,
+    stealth,
+    resistance,
+    stage_speed,
+    transmission,
+    level,
+    neutered,
+  } = symptom;
+
+  const thresholds = symptom.threshold_desc || {};
+
+  return (
+    <Section
+      title={name}
+      level={2}
+      buttons={!!neutered && (
+        <Box
+          bold
+          color="bad" >
+          Neutered
+        </Box>
+      )}
+    >
+      <Grid>
+        <Grid.Column size={2}>
+          {desc}
+        </Grid.Column>
+        <Grid.Column>
+          <LabeledList>
+            <LabeledList.Item label="Level">
+              {level}
+            </LabeledList.Item>
+            <LabeledList.Item label="Resistance">
+              {resistance}
+            </LabeledList.Item>
+            <LabeledList.Item label="Stealth">
+              {stealth}
+            </LabeledList.Item>
+            <LabeledList.Item label="Stage Speed">
+              {stage_speed}
+            </LabeledList.Item>
+            <LabeledList.Item label="Transmission">
+              {transmission}
+            </LabeledList.Item>
+          </LabeledList>
+        </Grid.Column>
+      </Grid>
+      {thresholds !== {} && (
```

Comment:
You can't compare objects like this. Equality compares object identities (pointers), not contents. One way you could do it cleanly, is to convert it to a list first (using the same `map()`, but near your data destructurings), then check `length` property on it.

Something like:
```js
const thresholds = map((desc, label) => ({ desc, label }))(data.thresholds || {});
```

As a bonus, you get a clean array to work with. But ideally, of course, you want backend to send clean arrays:

```dm
threshold_descs = list(
  list(
    "label" = "Resistance 8"
    "desc" = "Additionally regenerates lost blood."
  ),
  ...
)
```